internal/layers/layers.go

```diff
@@ -411,15 +414,16 @@ func addLCOWLayer(ctx context.Context, vm *uvm.UtilityVM, layerPath string) (uvm
 
 	sm, err := vm.SCSIManager.Add(
 		ctx,
-		&scsi.AttachConfig{Path: layerPath, ReadOnly: true, Type: scsi.AttachmentTypeVirtualDisk},
-		&scsi.MountConfig{ReadOnly: true, Verity: scsi.ReadVerityInfo(ctx, layerPath), Options: []string{"ro"}},
+		&scsi.AttachConfig{Path: layer.VHDPath, ReadOnly: true, Type: scsi.AttachmentTypeVirtualDisk},
+		&scsi.MountConfig{ReadOnly: true, Verity: scsi.ReadVerityInfo(ctx, layer.VHDPath), Options: []string{"ro"}},
```

Comment:
I think this is missing the layer partition

Reply:
Good catch!